compute_endpoint/globus_compute_endpoint/endpoint/config/utils.py

```diff
@@ -230,24 +230,36 @@ def _shell_escape_filter(val):
     return json.dumps(shlex.quote(loaded))
 
 
-def render_config_user_template(endpoint_dir: pathlib.Path, user_opts: dict) -> str:
-    import jinja2  # Only load package when called by EP manager
+def load_user_config_template(endpoint_dir: pathlib.Path) -> tuple[str, dict | None]:
     from globus_compute_endpoint.endpoint.endpoint import Endpoint
-    from jinja2.sandbox import SandboxedEnvironment
+
+    user_config_path = Endpoint.user_config_template_path(endpoint_dir)
+    template_str = _read_config_yaml(user_config_path)
 
     user_config_schema = load_user_config_schema(endpoint_dir)
+
+    return template_str, user_config_schema
+
+
+def render_config_user_template(
```

Comment:
Assuming that jinja rendering could be unsafe, please add a note that this method should be called after we drop privileges.